src/cls/2pc_queue/cls_2pc_queue_types.h

```diff
@@ -54,6 +59,7 @@ struct cls_2pc_urgent_data
     DECODE_START(1, bl);
     decode(reserved_size, bl);
     decode(last_id, bl);
+    decode(committed_entries, bl);
```

Comment:
new entries have to be at the end (so that old RGWs could ignore them)

Reply:
Ack!